solutions/usaco-531.mdx

```diff
@@ -121,4 +121,59 @@ int main()
 
 </CPPSection>
 
+<JavaSection>
```

Comment:
this is prim's and the c++ sol is kruskal's, so put this in a separate language section (and maybe add headers to the sections)